docs/getting-started-guides/kubeadm.md

```diff
@@ -149,28 +183,12 @@ For example:
 
 A few seconds later, you should notice that running `kubectl get nodes` on the master shows a cluster with as many machines as you created.
```

Comment:
Also, should the user validate the number of running nodes and cluster-info with `kubectl`?